tidy3d/components/monitor.py

```diff
@@ -673,6 +693,17 @@ class ModeSolverMonitor(AbstractModeMonitor):
         "primal grid nodes).",
     )
 
+    @pydantic.root_validator(skip_on_failure=True)
+    def deprecation_warning(cls, values):
```

Comment:
we should ensure that this change is reflected in our notebooks after this?

Reply:
Yeah, but I think there's only one notebook that uses a ModeSolverMonitor so should be easy.